src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -12,6 +12,7 @@ $pf-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl"
 
   // Item
   --pf-c-toolbar__item--Display: block;
+  --pf-c-toolbar__item--MinWidth--base: 0;
```

Comment:
I believe the `min-width` of a flex item is `auto` by default, which will evaluate to `min-content` in a horizontal flex row. I think `0` is almost always better, but if it was undefined before, changing it to `0` could cause a layout change.